website/source/docs/concepts/tokens.html.md

```diff
@@ -42,14 +42,61 @@ Finally, the `-policy` flag can be used to set the policies associated
-functioning, Vault will revoke it.
+Every non-root token has a time-to-live (TTL) associated with it, which is
+the number of seconds the token is valid for from this moment in time.
+The TTL is constantly decreasing, and Vault will revoke the token when the TTL hits 0.
+You can find out how much of a token's TTL remains using the `vault token-lookup` command.
 
-When a token is revoked, any leases associated with the token will be revoked
-as well, even if the TTLs on the individual leases are not yet up. For example,
-if a user requests AWS access keys, after the token expires the AWS access keys
-will also be revoked. In order to avoid your token being revoked, the `vault
-token-renew` command should be periodically used to renew the token.
+When a token is revoked, any leases associated with the token will also be revoked,
+even if the TTLs on the individual leases are not yet up. For example,
+if a developer uses a token to generate AWS access keys, the AWS access keys
+will be revoked when the token is revoked. The developer will then need to
+create a new token, and use it to request a new set of AWS access keys.
+
+To stop your token being revoked, you should call the `vault
+token-renew` command periodically, which will attempt to extend the token's TTL.
+You can specify _the new TTL_ using the `increment` argument. If this is
+not specified, then Vault will use the default TTL specified by the
+`auth/token` mount, or the global [`default_lease_ttl`](/docs/config/index.html#default_lease_ttl).
+
+**NOTE:** Vault enforces an upper bound on how long non-root tokens can exist.
+This bound is called the "max lease TTL", and is measured in hours from
+the moment the token was created. Vault will not allow a token to be renewed
+beyond this point. If you attempt to do so, Vault will cap the TTL at a value
+that will not cause the token to exist beyond the max lease TTL.
+
+You can detect if the token has hit the max lease TTL by comparing the token's
+TTL after renewal to the `increment` you provided when renewing it. If the TTL
+is less than half of the requested `increment` then you should generate a new token.
+
+As an example, suppose Vault's "max lease TTL" has been set to 10 hours,
+and you have a token that was created 6 hours ago.
+If you were to renew this token with an `increment` of 9 hours,
+that would cause the token to exist for longer than the "max lease TTL" of
+10 hours (`6 + 9 = 15`), so instead Vault will ignore `increment` and
+cap the TTL at 4 hours (`10 - 6 = 4`). 4 hours is less than half of the requested
+`increment`, so you should generate a new token and prepare for the fact that
+Vault will soon be revoking the old one.
+
+You can configure this cap by changing the system's
+[`max_lease_ttl`](/docs/config/index.html#max_lease_ttl) (which will affect
+all backends, such as AWS, PKI etc.), but the preferred method is to tune
+the `auth/token` mount itself:
```

Comment:
It may be worth noting that the mount value takes precedent over -- and can be longer than -- the system value. This is unlike configured values in supporting backends, where the value a backend asks for can never be higher than the mount max (or, if it's not set, the system default max). The mount max trumps all.